CMakeLists.txt

```diff
@@ -139,19 +147,25 @@ endif()
 
 if(CMAKE_COMPILER_IS_MSVC)
   add_definitions("-DBOOST_ALL_NO_LIB -D_SCL_SECURE_NO_WARNINGS -D_CRT_SECURE_NO_WARNINGS -DNOMINMAX ${SSE_DEFINITIONS}")
-  
+
   if("${CMAKE_CXX_FLAGS}" STREQUAL "${CMAKE_CXX_FLAGS_DEFAULT}")
+    
     string(APPEND CMAKE_CXX_FLAGS " /fp:precise ${SSE_FLAGS} ${AVX_FLAGS}")
+    set(CMAKE_CUDA_FLAGS "${CMAKE_CUDA_FLAGS} -Xcompiler=/bigobj")
+
+    set(PCL_USE_GLOBAL_OPTIMIZATION TRUE)
+    if(CUDA_FOUND)
+      if(${CUDA_VERSION_STRING} VERSION_GREATER_EQUAL "10.0" AND ${CUDA_VERSION_STRING} VERSION_LESS "12.0")
+        set(USE_GLOBAL_OPTIMIZATION FALSE)
```

Comment:
Is this meant to be `PCL_USE_GLOBAL_OPTIMIZATION`? Should the user be allowed to change `PCL_USE_GLOBAL_OPTIMIZATION`, or does that not make sense?

Reply:
Yes, indeed it was. Thanks